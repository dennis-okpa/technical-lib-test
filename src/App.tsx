import React from 'react'
import Library from './components/Pages/Library'

export const App = () => <Library />
