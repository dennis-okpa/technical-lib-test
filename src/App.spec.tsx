import React from 'react'
import { App } from './App'
import { render, getByText } from '@testing-library/react'

describe('The App component', () => {
  describe('when it renders', () => {
    let container: HTMLElement

    beforeEach(() => {
      ({ container } = render(<App />))
    });
    
    it('should match snapshot', () => {
      expect(container).toMatchSnapshot()
    });

    it('should contain expected text', () => {
      const mainText = getByText(container, 'Hello World')

      expect(mainText).toBeInTheDocument()
    })
  })
})
