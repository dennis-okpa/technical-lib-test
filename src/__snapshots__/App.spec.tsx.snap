// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The App component when it renders should match snapshot 1`] = `
<div>
  <div>
     Hello World 
  </div>
</div>
`;
